src/main/java/org/zaproxy/zap/extension/jwt/attacks/CommonPasswordAttack.java

```diff
@@ -0,0 +1,75 @@
+package org.zaproxy.zap.extension.jwt.attacks;
+
+import java.util.ArrayList;
+import java.util.List;
+import org.apache.log4j.Logger;
+import org.parosproxy.paros.core.scanner.Alert;
+import org.zaproxy.zap.extension.jwt.JWTConfiguration;
+import org.zaproxy.zap.extension.jwt.JWTHolder;
+import org.zaproxy.zap.extension.jwt.exception.JWTException;
+import org.zaproxy.zap.extension.jwt.utils.JWTConstants;
+import org.zaproxy.zap.extension.jwt.utils.JWTUtils;
+import org.zaproxy.zap.extension.jwt.utils.VulnerabilityType;
+
+/**
+ * There are publicly available well known JWT HMac Secrets and This attack checks if JWT is signed
+ * using weak well known secret.
+ *
+ * <p>Special thanks to <a
+ * href="https://lab.wallarm.com/340-weak-jwt-secrets-you-should-check-in-your-code/">Wallarm.com</a>
+ * for collating the list of such weak secrets and making them as opensource.
+ *
+ * <p>For knowing all the secrets please visit: <a
+ * href="https://raw.githubusercontent.com/wallarm/jwt-secrets/master/jwt.secrets.list">Weak
+ * Publicly known HMac Secrets</a>
+ *
+ * @author KSASAN preetkaran20@gmail.com
+ */
+public class CommonPasswordAttack implements JWTAttack {
```

Comment:
Should be added as part of Bruteforce attack or some separate attack or running under a configuration flag in Options tab.

Reply:
Need to think more on this. For now just added to check if everything works properly or not.